blueocean-pipeline-editor/src/main/js/jenkins-js-extension.yaml

```diff
@@ -0,0 +1,7 @@
+# Extensions in this plugin
+# NB: "component" currently maps to modules, not "symbols" so make sure to "export default"
+# WARNING: If you change this you'll have to change io.jenkins.blueocean.jsextensions.JenkinsJSExtensionsTest as well :(
```

Comment:
A big dirty warning is given if there's no default export, so I don't think there's a need for that comment anymore.